src/fsharp/ast.fs

```diff
@@ -1742,7 +1755,7 @@ let mkSynDotBrackSeqSliceGet  m mDot arr (argslist:list<SynIndexerArg>) =
                        | SynIndexerArg.One x -> yield x
                        | _ -> () ]
     if notsliced.Length = argslist.Length then
-        SynExpr.DotIndexedGet(arr,[SynIndexerArg.One (SynExpr.Tuple(notsliced,[],unionRanges (Seq.head notsliced).Range (Seq.last notsliced).Range))],mDot,m)
```

Comment:
The problem here is that although we create a proto version of FSharp.Build.dll, for some reason we don't create a proto version of FSharp.Core. This should probably be corrected.

The internal official build does create a proto version of FSharp.Core, and references that when compiling proto ver of FSharp.Compiler.  Right now OSS is just referencing the LKG FSharp.Core here, which is missing a lot of stuff.